deploy-manage/manage-connectors.md

```diff
@@ -2,25 +2,90 @@
+
+:::{image} ../images/kibana-connector-delete.png
+:alt: Deleting connectors individually or in bulk
+:class: screenshot
+:::
+
+::::{note}
+You can delete a connector even if there are still actions referencing it. When this happens the action will fail to run and errors appear in the {{kib}} logs.
+
+::::
+
+## Creating a new connector [creating-new-connector]
+
+New connectors can be created with the **Create connector** button, which guides you to select the type of connector and configure its properties. For a full list of available connectors, see [Available connectors](asciidocalypse://docs/kibana/docs/reference/connectors-kibana/connectors-kibana.md).
+
+::::{note}
+Some connector types are paid commercial features, while others are free. For a comparison of the Elastic subscription levels, go to [the subscription page](https://www.elastic.co/subscriptions).
+::::
+
+:::{image} ../images/kibana-connector-select-type.png
+:alt: Connector select type
+:class: screenshot
+:width: 75%
+:::
+
+After you create a connector, it is available for use any time you set up an action in the current space.
+
+::::{tip}
+**Preconfigured connectors and the Terraform providers are not available in serverless projects.**
+
+For out-of-the-box and standardized connectors, refer to [preconfigured connectors](https://www.elastic.co/guide/en/kibana/current/pre-configured-connectors.html). 
+
+You can also manage connectors as resources with the [Elasticstack provider](https://registry.terraform.io/providers/elastic/elasticstack/latest) for Terraform. For more details, refer to the [elasticstack_kibana_action_connector](https://registry.terraform.io/providers/elastic/elasticstack/latest/docs/resources/kibana_action_connector) resource.
+::::
+
+## Importing and exporting connectors [importing-and-exporting-connectors]
+
+To import and export connectors, use the [Saved Objects Management UI](/explore-analyze/find-and-organize/saved-objects.md).
+
+If a connector is missing sensitive information after the import, a **Fix** button appears in **{{connectors-ui}}**.
+
+:::{image} ../images/kibana-connectors-with-missing-secrets.png
+:alt: Connectors with missing secrets
+:class: screenshot
+:::
+
+## Monitoring connectors [monitoring-connectors]
+
+For all deployment types except for serverless projects, you can use the [Task Manager health API](../deploy-manage/monitor/kibana-task-manager-health-monitoring.md) to help you understand the performance of all tasks in your environment. However, if connectors fail to run, they will report as successful to Task Manager. The failure stats will not accurately depict the performance of connectors.
+
+Additionally, you can use the [console](/explore-analyze/query-filter/tools/console.md) to query the [Event log index](../explore-analyze/alerts-cases/alerts/event-log-index.md) to gather information on connector successes and failures on any type of deployment.
```

Comment:
I'd swap this to put the "works for everyone" stuff first and the "only for some people" stuff second

also, you don't need to use the console to query an index ... you can query an index from anywhere

```suggestion
You can query the [Event log index](/explore-analyze/alerts-cases/alerts/event-log-index.md) to gather information on connector successes and failures.

If you're using {{stack}}, then you can also use the [Task Manager health API](/deploy-manage/monitor/kibana-task-manager-health-monitoring.md) to monitor connector performance. However, if connectors fail to run, they will report as successful to Task Manager. The failure stats will not accurately depict connector failures.
```